lib/saml_idp/request.rb

```diff
@@ -150,19 +150,7 @@ def matching_cert
       return nil unless signed?
 
       Array(service_provider.certs).find do |cert|
-        document.valid_signature?(
-          fingerprint(cert),
-          options.merge(cert:)
-        )
-      end
-    end
-
-    def sha256_validation_matching_cert
-      return nil unless signed?
-
-      Array(service_provider.certs).find do |cert|
-        document.valid_sig_with_sha256?(cert, options)
-      rescue SamlIdp::XMLSecurity::SignedDocument::ValidationError
+        document.valid_signature?(cert, options.merge(cert:))
```

Comment:
we probably no longer need to pass in `cert:` as an option right?

Reply:
oh, yes, i think so, since we're no longer using it as a backup

Reply:
updated in https://github.com/18F/saml_idp/pull/130/commits/243163a76d36ecaa037f9138836059ffca412014